packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -337,6 +338,42 @@ export default class TinyMCE extends Component {
 		}
 	}
 
+	/**
+	 * Replaces non breaking spaces inserted by TinyMCE on the space key by
+	 * a normal space.
+	 *
+	 * @param  {SyntheticEvent} event Synthetic key up event.
+	 */
+	onKeyUp( event ) {
+		if ( event.shiftKey || event.keyCode !== SPACE ) {
```

Comment:
What is it about the shift key that matters?

Reply:
It inserts a non breaking space. :) I'll comment.